src/routes/index.tsx

```diff
@@ -29,6 +32,16 @@ export default () => {
           </Route>
           {/* <-- INJECT ROUTE --> */}
 
+          <AuthContextProvider>
+            <Route exact path={ROUTES.matrixLogin}>
+              <Login />
+            </Route>
+
+            <AuthRoute exact path={ROUTES.matrixPersonal}>
+              <div>aaa</div>
```

Comment:
What's this `aaa`?

Reply:
It's added only for checking if re-routing is working. It will be replaced with real Component in next PR